Source/WebCore/rendering/RenderFragmentContainer.cpp

```diff
@@ -156,26 +157,27 @@ LayoutPoint RenderFragmentContainer::fragmentedFlowPortionLocation() const
 LayoutRect RenderFragmentContainer::overflowRectForFragmentedFlowPortion(const LayoutRect& fragmentedFlowPortionRect, bool isFirstPortion, bool isLastPortion) const
 {
     ASSERT(isValid());
-    if (shouldClipFragmentedFlowContent())
-        return fragmentedFlowPortionRect;
 
-    LayoutRect fragmentedFlowOverflow = visualOverflowRectForBox(*m_fragmentedFlow);
-    LayoutRect clipRect;
+    auto renderableInfiniteRect = [] {
+        // Return a infinite-like rect whose values are such that, when converted to float pixel values, they can reasonably represent device pixels.
+        return LayoutRect(LayoutUnit::nearlyMin() / 32, LayoutUnit::nearlyMin() / 32, LayoutUnit::nearlyMax() / 16, LayoutUnit::nearlyMax() / 16);
+    }();
```

Comment:
any particular reason for not using LayoutRect's infiniteRect?

Reply:
It leads to WPT failure - css/css-break/out-of-flow-in-multicolumn-018.htm.

Reply:
Let me debug this.

Reply:
I am going to defer to @smfr on this, but 1, it does not scale well 2, technically we now return a non-infinite rect even when there's no clipping at all (so comparing to LayoutRect::infiniteRect() will return false)